File: output/CodeGen/avatar/Java/Python/codeforces_450_A.py
def main(args):
    sc = Scanner(System.in)
    n = sc.nextInt()
    m = sc.nextInt()
    q1 = []
    q2 = []
    for i in range(1, n+1):
        q1.append(sc.nextInt())
        q2.append(i)
    while q1:
        if q1[0] <= m:
            q1.pop(0)
            q2.pop(0)
            ans = q2.pop(0)
        else:
            if q1[0] > m:
                q1.pop(0)
                q2.pop(0)
                q1.append(q1[0]-m)
                q2.append(q2[0])
    print(ans)

File: output/StarCoder/avatar/Python/Java/codeforces_450_A.java
import java.util.*;
public class codeforces_450_A {
    public static void main ( String [ ] args )
    {
        Scanner sc = new Scanner ( System.in ) ;
        int n = sc.nextInt ( ) ;
        int m = sc.nextInt ( ) ;
        int [ ] l = new int [ n ] ;
        for ( int i = 0 ; i < n ; i++ )
            l [ i ] = sc.nextInt ( ) ;
        int [ ] l2 = new int [ n ] ;
        int mx = 0 ;
        for ( int i = 0 ; i < n ; i++ )
        {
            if ( l [ i ] % m == 0 )
                l2 [ i ] = l [ i ] / m ;
            else
                l2 [ i ] = l [ i ] / m + 1 ;
            if ( l2 [ i ] > mx )
            {
                mx = l2 [ i ] ;
                ind = i ;
            }
        }
        System.out.println ( ind + 1 ) ;
    }
}